javascript-timers/timeout.js

```diff
@@ -0,0 +1,6 @@
+function message() {
+  document.querySelector('.message').innerText = 'Hello There';
```

Comment:
In general, it's safer and faster to set `textContent` than it is to set `innerText`.